ui/src/components/Settings/Notifications/index.tsx

```diff
@@ -0,0 +1,192 @@
+import {
+  DocumentAddIcon,
+  PlusCircleIcon,
+  TrashIcon,
+} from '@heroicons/react/solid'
+import { useContext, useEffect, useState } from 'react'
+import { useToasts } from 'react-toast-notifications'
+import SettingsContext from '../../../contexts/settings-context'
+import GetApiHandler, { DeleteApiHandler } from '../../../utils/ApiHandler'
+import Button from '../../Common/Button'
+import CreateNotificationModal, {
+  AgentConfiguration,
+} from './CreateNotificationModal'
+import CachedImage from '../../Common/CachedImage'
+
+const NotificationSettings = () => {
+  const settingsCtx = useContext(SettingsContext)
+  const [addModalActive, setAddModalActive] = useState(false)
+  const [configurations, setConfigurations] = useState<AgentConfiguration[]>()
+  const [editConfig, setEditConfig] = useState<AgentConfiguration>()
+  const { addToast } = useToasts()
+
+  const basePath = process.env.NEXT_PUBLIC_BASE_PATH ?? ''
+
+  useEffect(() => {
+    document.title = 'Maintainerr - Settings - Notifications'
+    GetApiHandler('/notifications/configurations').then((configs) =>
+      setConfigurations(configs),
+    )
+  }, [])
+
+  useEffect(() => {
+    GetApiHandler('/notifications/configurations').then((configs) =>
+      setConfigurations(configs),
+    )
+  }, [addModalActive])
+
+  const doEdit = (id: number) => {
+    const config = configurations?.find((c) => c.id === id)
+
+    setEditConfig(config)
+    setAddModalActive(!addModalActive)
+  }
+
+  function confirmedDelete(id: any) {
+    DeleteApiHandler(`/notifications/configuration/${id}`).then(() => {
+      setConfigurations(configurations?.filter((c) => c.id !== id))
+    })
+  }
+
+  return (
+    <div className="h-full w-full">
+      <div className="section-settings h-full w-full">
```

Comment:
Just use the Tailwind classes directly

```suggestion
      <div className="mb-5 mt-6 text-white h-full w-full">
```